lib/interpreters/periodic_visits.rb

```diff
@@ -513,24 +513,24 @@ def generate_routes(vrp)
     end
 
     def generate_rests(vehicle, day_index, rests_durations)
-      new_rests = vehicle.rests.collect{ |rest|
+      vehicle.rests.collect{ |rest|
         new_rest = nil
-        if rest.timewindows.empty? || rest.timewindows.any?{ |timewindow| timewindow[:day_index] == day_index }
+        if rest.timewindows.empty? || rest.timewindows.any?{ |timewindow| timewindow.day_index.nil? || timewindow.day_index == day_index }
```

Comment:
may this is missing a modulo? because `timewindow.day_index` (i don't know if this is true for `day_index`)

Reply:
We need a test which passes through this function.